critiquebrainz/ws/review/views.py

```diff
@@ -337,6 +337,7 @@ def review_list_handler():
     :query limit: results limit, min is 0, max is 50, default is 50 **(optional)**
     :query offset: result offset, default is 0 **(optional)**
     :query language: language code (ISO 639-1) **(optional)**
+    :query review_type: ``all`` or ``text`` or ``rating`` **(optional)**
```

Comment:
I think that we can be a bit more descriptive in this documentation. At the moment, it's unclear when it should be used. How about something like:

    :query review_type: ``review`` or ``rating`` (optional). If set, only return reviews which have a text review, or a rating.

I think that we can remove the `all` option. This is the same as not adding the parameter. I also think that we should rename text-> review

Check also the documentation for "language code", I think this should be a little bit clearer to explain that it will filter only reviews with the given language code.
See also the documentation for "entity_id" - this no longer refers to a release group.
The documentation for entity_type should have a link to a list in the documentation that shows a list of all reviewable entities. We do this in listenbrainz:
https://github.com/metabrainz/listenbrainz-server/blob/e39c2f27e19867fa1293612f5b4c9b9e90b01f62/docs/dev/json.rst#L35
https://github.com/metabrainz/listenbrainz-server/blob/684e28863f209baa79c3e7e571b3f56b94f9b62f/docs/dev/api.rst#L119
(I think that github won't link you to the correct line for these files, but you can open them up yourself and go to that line)

Reply:
Done